File: ANG/src/app/view-model/quiz-master-all-data.ts
export interface OptionAllData {
    id: number;
    questionMasterId: number;
    optionText: string;
    isCorrectAnswer: boolean;
}

export interface QuestionAllData {
    id: number;
    quizMasterId: number;
    questionText: string;
    options: Array<OptionAllData>;
}

export interface QuizMasterAllData {
    id: number;
    subject: string;
    questions : Array<QuestionAllData>;
}

File: SB/go-poll/src/main/kotlin/com/gopoll/model/QuizMasterAllData.kt
package com.gopoll.model

data class OptionAllData(val id: Long, val questionMasterId: Long,
                         val optionText: String, val isCorrectAnswer: Boolean)

data class QuestionAllData(val id: Long, val quizMasterId: Long,
                           val questionText: String, val options:
                           List<OptionAllData>)

data class QuizMasterAllData(val id: Long, val subject: String, val questions:
List<QuestionAllData>)